drivers/modem/ublox-sara-r4.c

```diff
@@ -799,6 +994,25 @@ static void modem_reset(void)
 		goto error;
 	}
 
+#if defined(CONFIG_MODEM_UBLOX_SARA_AUTODETECT_APN)
+	/* autodetect APN from IMSI */
+	char cmd[64];
```

Comment:
Instead of this hard coded value, I think something like below would be better (user does not have to count the chars itself as the compiler is doing that for you). The APN name just needs to be the longest name possible.
```
char cmd[sizeof("AT+CGDCONT=1,'IP','012345678901234567890123456789012'")];
```

Reply:
good point! I changed it to use the correct (max) length now.